<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Deposit</class>
 <widget class="QWidget" name="Deposit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>686</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Rate</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_rate"/>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="comboBox_frequency">
       <item>
        <property name="text">
         <string>every day</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>every month</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>every quater</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>every year</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Timeframe</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEdit_tax"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Frequency</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Amount</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_amount"/>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboBox_timeframe">
       <item>
        <property name="text">
         <string>1 month</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3 month</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>6 month</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>9 month</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1 year</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2 year</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3 year</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4 year</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5 year</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>10 year</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>20 year</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Tax Rate</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="2">
      <widget class="QDateEdit" name="dateEdit_elim">
       <property name="dateTime">
        <datetime>
         <hour>0</hour>
         <minute>0</minute>
         <second>0</second>
         <year>2024</year>
         <month>1</month>
         <day>1</day>
        </datetime>
       </property>
       <property name="maximumDateTime">
        <datetime>
         <hour>20</hour>
         <minute>59</minute>
         <second>59</second>
         <year>9999</year>
         <month>1</month>
         <day>1</day>
        </datetime>
       </property>
       <property name="minimumDate">
        <date>
         <year>2024</year>
         <month>1</month>
         <day>1</day>
        </date>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLineEdit" name="lineEdit_supp"/>
     </item>
     <item row="2" column="2">
      <widget class="QDateEdit" name="dateEdit_supp">
       <property name="dateTime">
        <datetime>
         <hour>0</hour>
         <minute>0</minute>
         <second>0</second>
         <year>2024</year>
         <month>1</month>
         <day>1</day>
        </datetime>
       </property>
       <property name="minimumDateTime">
        <datetime>
         <hour>0</hour>
         <minute>0</minute>
         <second>0</second>
         <year>2024</year>
         <month>1</month>
         <day>1</day>
        </datetime>
       </property>
       <property name="minimumDate">
        <date>
         <year>2024</year>
         <month>1</month>
         <day>1</day>
        </date>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="comboBox_supp">
       <item>
        <property name="text">
         <string>one-time</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>once a month</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>once a quater</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>once a year</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="checkBox_supplementions">
       <property name="text">
        <string>Supplementions</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QCheckBox" name="checkBox_eliminations">
       <property name="text">
        <string>Eliminations</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QCheckBox" name="checkBox_capitalization">
       <property name="text">
        <string>Capitalization</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QLineEdit" name="lineEdit_elim"/>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="comboBox_elim">
       <item>
        <property name="text">
         <string>one-time</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>once a month</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>once a quater</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>once a year</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="0" colspan="4">
      <widget class="QPushButton" name="pushButton_calculate">
       <property name="text">
        <string>Calculate</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1" rowspan="2">
    <layout class="QGridLayout" name="gridLayout_3">
     <property name="horizontalSpacing">
      <number>70</number>
     </property>
     <item row="1" column="1">
      <widget class="QLabel" name="label_procent">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Tax</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_tax">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Total</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Accured interest</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="label_total">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Result</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
